sebastian-rendon/tarea-2/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="Mock Magazine Cover about Oscar Piastri"
+    />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Magazine</title>
+  </head>
+  <body>
+    <main class="main__magazine">
+      <span
```

Comment:
@s-rendong, research about the use of date tag

Reply:
fixed with https://github.com/emolinaab/bootcamp-001/pull/88/commits/8393c4295e96ea5b6bb1fd2804e2903da42f3180